### ISSUE TITLE
Cannot resolve host address, SIGUSR1 received, process restarting

### ISSUE THREAD

**Issue by Moose88** [haugene/docker-transmission-openvpn#1598]
**Before creating this issue I have:**

- [X] Searched for [similar issues](https://github.com/haugene/docker-transmission-openvpn/issues)
- [x] [Read the documentation](https://haugene.github.io/docker-transmission-openvpn/). Especially the troubleshooting section
- [X] Tried to add as much relevant information to the issue as possible

**Describe the problem**
For about a week now, I added a few series to transmission, and they'll start off just fine, but after a day, they then have 0 upload/download. I checked the logs and got what I pasted below. I've changed the OPENVPN_CONFIG multiple times, and set it to the following in my latest change: ca_vancouver,ca_ontario,ca_montreal. I'm still getting the same.

**Logs**
```
2020-12-09 00:17:48 | stdout | Wed Dec  9 00:17:48 UTC 2020: bindPort error
-- | -- | --
2020-12-09 00:17:48 | stdout | curl: (28) Connection timed out after 15001 milliseconds
2020-12-09 00:17:18 | stdout | curl: (28) Connection timed out after 15001 milliseconds
2020-12-09 00:16:48 | stdout | curl: (28) Connection timed out after 15000 milliseconds
2020-12-09 00:16:18 | stdout | curl: (28) Connection timed out after 15000 milliseconds
2020-12-09 00:16:00 | stdout | Wed Dec  9 00:16:00 2020 SIGUSR1[soft,init_instance] received, process restarting
2020-12-09 00:16:00 | stdout | Wed Dec  9 00:16:00 2020 Could not determine IPv4/IPv6 protocol
2020-12-09 00:16:00 | stdout | Wed Dec  9 00:16:00 2020 RESOLVE: Cannot resolve host address: ca-montreal.privacy.network:1198 (Try again)
2020-12-09 00:15:55 | stdout | Wed Dec  9 00:15:55 2020 RESOLVE: Cannot resolve host address: ca-montreal.privacy.network:1198 (Try again)
2020-12-09 00:15:50 | stdout | Wed Dec  9 00:15:50 2020 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
2020-12-09 00:15:48 | stdout | curl: (28) Connection timed out after 15001 milliseconds
2020-12-09 00:15:18 | stdout | curl: (28) Connection timed out after 15001 milliseconds
2020-12-09 00:10:50 | stdout | Wed Dec  9 00:10:50 2020 SIGUSR1[soft,init_instance] received, process restarting
2020-12-09 00:10:50 | stdout | Wed Dec  9 00:10:50 2020 Could not determine IPv4/IPv6 protocol
2020-12-09 00:10:50 | stdout | Wed Dec  9 00:10:50 2020 RESOLVE: Cannot resolve host address: ca-montreal.privacy.network:1198 (Try again)
2020-12-09 00:10:45 | stdout | Wed Dec  9 00:10:45 2020 RESOLVE: Cannot resolve host address: ca-montreal.privacy.network:1198 (Try again)
2020-12-09 00:10:40 | stdout | Wed Dec  9 00:10:40 2020 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
2020-12-09 00:05:40 | stdout | Wed Dec  9 00:05:40 2020 SIGUSR1[soft,init_instance] received, process restarting
2020-12-09 00:05:40 | stdout | Wed Dec  9 00:05:40 2020 Could not determine IPv4/IPv6 protocol
2020-12-09 00:05:40 | stdout | Wed Dec  9 00:05:40 2020 RESOLVE: Cannot resolve host address: ca-montreal.privacy.network:1198 (Try again)
2020-12-09 00:05:35 | stdout | Wed Dec  9 00:05:35 2020 RESOLVE: Cannot resolve host address: ca-montreal.privacy.network:1198 (Try again)
2020-12-09 00:05:30 | stdout | Wed Dec  9 00:05:30 2020 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
2020-12-09 00:00:30 | stdout | Wed Dec  9 00:00:30 2020 SIGUSR1[soft,init_instance] received, process restarting
2020-12-09 00:00:30 | stdout | Wed Dec  9 00:00:30 2020 Could not determine IPv4/IPv6 protocol
2020-12-09 00:00:30 | stdout | Wed Dec  9 00:00:30 2020 RESOLVE: Cannot resolve host address: ca-montreal.privacy.network:1198 (Try again)
2020-12-09 00:00:25 | stdout | Wed Dec  9 00:00:25 2020 RESOLVE: Cannot resolve host address: ca-montreal.privacy.network:1198 (Try again)
2020-12-09 00:00:20 | stdout | Wed Dec  9 00:00:20 2020 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
2020-12-08 23:55:20 | stdout | Tue Dec  8 23:55:20 2020 SIGUSR1[soft,init_instance] received, process restarting
2020-12-08 23:55:20 | stdout | Tue Dec  8 23:55:20 2020 Could not determine IPv4/IPv6 protocol
2020-12-08 23:55:20 | stdout | Tue Dec  8 23:55:20 2020 RESOLVE: Cannot resolve host address: ca-montreal.privacy.network:1198 (Try again)
2020-12-08 23:55:15 | stdout | Tue Dec  8 23:55:15 2020 RESOLVE: Cannot resolve host address: ca-montreal.privacy.network:1198 (Try again)
2020-12-08 23:55:10 | stdout | Tue Dec  8 23:55:10 2020 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
2020-12-08 23:50:10 | stdout | Tue Dec  8 23:50:10 2020 SIGUSR1[soft,init_instance] received, process restarting
2020-12-08 23:50:10 | stdout | Tue Dec  8 23:50:10 2020 Could not determine IPv4/IPv6 protocol
2020-12-08 23:50:10 | stdout | Tue Dec  8 23:50:10 2020 RESOLVE: Cannot resolve host address: ca-montreal.privacy.network:1198 (Try again)
2020-12-08 23:50:05 | stdout | Tue Dec  8 23:50:05 2020 RESOLVE: Cannot resolve host address: ca-montreal.privacy.network:1198 (Try again)
2020-12-08 23:50:00 | stdout | Tue Dec  8 23:50:00 2020 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
2020-12-08 23:45:03 | stdout | the has been a fatal_error
```

**Host system:**
Running on Synology NAS.

**Comment by haugene** [comment]
The container is apparently getting DNS errors. There are two log lines that is especially relevant here:

```
RESOLVE: Cannot resolve host address: ca-montreal.privacy.network:1198 (Try again)
```
:point_up: Ok, so your DNS resolving is gone. Doing one of [these suggestions](https://haugene.github.io/docker-transmission-openvpn/faq/#error_resolving_host_address) might remedy that. But.. As you say, the container starts off just fine. So what happens?

And then we have:
```
SIGUSR1[soft,init_instance] received, process restarting
```
:point_up: This is [described here](https://haugene.github.io/docker-transmission-openvpn/faq/#container_loses_connection_after_some_time). It's not always easy to know why a containers network falls out. I've been running this container for 5-6 years now and haven't had that problem. But still. There is a proposed solution in the docs. Test it and let me know how it goes :crossed_fingers:

**Comment by Moose88** [comment]
> The container is apparently getting DNS errors. There are two log lines that is especially relevant here:
> 
> ```
> RESOLVE: Cannot resolve host address: ca-montreal.privacy.network:1198 (Try again)
> ```
> 
> ☝️ Ok, so your DNS resolving is gone. Doing one of [these suggestions](https://haugene.github.io/docker-transmission-openvpn/faq/#error_resolving_host_address) might remedy that. But.. As you say, the container starts off just fine. So what happens?
> 
> And then we have:
> 
> ```
> SIGUSR1[soft,init_instance] received, process restarting
> ```
> 
> ☝️ This is [described here](https://haugene.github.io/docker-transmission-openvpn/faq/#container_loses_connection_after_some_time). It's not always easy to know why a containers network falls out. I've been running this container for 5-6 years now and haven't had that problem. But still. There is a proposed solution in the docs. Test it and let me know how it goes 🤞

Roger that, I'll see what I can find and provide feedback.

**Comment by haugene** [comment]
Oh yeah. And I see you're running PIA and getting the bindPort stuff. Hopefully I've just merged a fix for that in #1602 
You could try to just pull the latest image again and see if it's fixed for you in the new version. That would also be interesting for me to know, if that ultimately caused your container networking to fail somehow. But I'm not seeing how some failed curl calls can cause that.

**Comment by Moose88** [comment]
> Oh yeah. And I see you're running PIA and getting the bindPort stuff. Hopefully I've just merged a fix for that in #1602
> You could try to just pull the latest image again and see if it's fixed for you in the new version. That would also be interesting for me to know, if that ultimately caused your container networking to fail somehow. But I'm not seeing how some failed curl calls can cause that.

Looks like it was a combination of this and the DNS resolving. Upgraded to latest and set the suggestion and now I'm running clean. Appreciate the update!
